website/source/docs/commands/alloc/exec.html.md.erb

```diff
@@ -0,0 +1,68 @@
+---
+layout: "docs"
+page_title: "Commands: alloc exec"
+sidebar_current: "docs-commands-alloc-exec"
+description: >
+Stream the exec of a task.
+---
+
+# Command: alloc exec
+**Alias: `nomad exec`**
+
+The `alloc exec` command runs a command in a running allocation.
+
+## Usage
+
+```
+nomad alloc exec [options] <allocation> <command> [<args>...]
+```
+
+
+This command runs the passed command inside the environment of the given allocation and task.
+
+This command streams the exec of the given task in the allocation. If the
```

Comment:
>This command streams the exec of the given task in the allocation.
Should that be `This command streams the execution of the given task in the allocation`?